plugins/sam3dj.js

```diff
@@ -0,0 +1,343 @@
+/** sam3DJ | aka. Sammy's .3DJ (format implementation)
+ *  ==================================================
+ *  Author:     Sammy L. Koch (1Turtle)
+ *  Licence:    Apache-2.0 (see https://www.apache.org/licenses/LICENSE-2.0.txt)
+*/
+(function() {    
+    const pluginProperties = {
+        title: 'sam3DJ',
+        author: '1Turtle',
+        tags: ['Minecraft: Java Edition', 'format', '3dj'],

```

Comment:
Let's include "Exporter" in the tags. I haven't written guidelines for tags yet, but that is planned, and should help make the tags more consistent across plugins.